src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  comments.length !== 0 ? (
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))}
+    </div>
+  ) : (
+    <>
+      <hr />
+      <b data-cy="NoCommentsMessage">
+        No comments yet
+      </b>
+    </>
```

Comment:
Try putting this logic to the parent components
It is more logical. `CommentList` should only be responsible for rendering the list and you decide if you want to render it in parent component.
Maybe this way it will allow to deploy

Reply:
As for `<hr>` it's needed to add if we don't have any comments
![Annotation 2023-10-11 123833](https://github.com/mate-academy/react_static-list-of-posts-js/assets/109142598/34ee966a-9652-466d-8253-56cb3c7d9894)

Reply:
Ye, I know, I just like as a response to your mate